topics/sequence-analysis/tutorials/ncbi-fcs/tutorial.bib

```diff
@@ -0,0 +1,30 @@
+
+# This is the bibliography file for your tutorial.
+#
+# To add bibliography (bibtex) entries here, follow these steps:
+#  1) Find the DOI for the article you want to cite
+#  2) Go to https://doi2bib.org and fill in the DOI
+#  3) Copy the resulting bibtex entry into this file
+#
+# To cite the example below, in your tutorial.md file
+# use {% cite Batut2018 %}
+#
+# If you want to cite an online resourse (website etc)
+# you can use the 'online' format (see below)
+#
+# You can remove the examples below
+
+@article{Astashya2024,
+  doi = {10.1186/s13059-024-03198-7},
+  url = {https://doi.org/10.1186/s13059-024-03198-7},
+  year = {2024},
+  month = feb,
+  publisher = {BioMed Central},
+  volume = {25},
+  number = {60},
+  author = {Astashyn A, Tvedte ES, Sweeney D, Sapojnikov V, Bouk N, Joukov V, Mozes E,
```

Comment:
> Need help with embedded citation Astashya2024 . Does author format need to be changed?

This looks fine, but author parsing is difficult. A lot of our bib entries seem to use `and` to separate authors, which is less ambiguous. I can recommend just pulling it from doi2bib:
https://www.doi2bib.org/bib/10.1186/s13059-024-03198-7